kahuna/public/stylesheets/main.css

```diff
@@ -518,6 +518,15 @@ input.search-query__input {
     display: none;
 }
 
+.file-uploader__select-files {
+    padding: 5px 10px;
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
It would, but the more rules the more you'll have to remember what scope (wrong term) you're in when looking at a rule - a separate file would make that clearer. No strong opinions either way and probably premature optimisation at this point.